### DISCUSSION TITLE
Automatically resend on SMPT 454

### DISCUSSION THREAD

**Discussion by willedstrom** [PHPMailer/PHPMailer#2493] [Q&A]
Working on fixing an occasional "454 Temporary service failure" using AWS.

Amazon Docs - https://docs.aws.amazon.com/ses/latest/DeveloperGuide/troubleshoot-smtp.html
Forum post suggesting that the appropriate response is being sent: https://forums.aws.amazon.com/thread.jspa?threadID=290433

Is there any way to make PHPMailer automatically resend in this case?

**Comment by Synchro** [comment]
No, that's an application level thing. Save your message details and generate it again later. If you want something to queue and retry automatically, use a local mail server because that's exactly what they are designed to do.

**Reply by willedstrom** [reply]
Ok will do, thanks for your prompt response.
